.travis.yml

```diff
@@ -2,17 +2,14 @@ language: ruby
 cache: bundler
 sudo: false
 rvm:
-  - 2.2
```

Comment:
**[IMO]** 2.2 and 2.3 are removed from build matrix.

This is soft deprecate? (build matrix is dropped, but users are allowed to install `rspec-parameterized` in ruby 2.2 and 2.3 )
or this is hard deprecate ? (we stop support in ruby 2.2 and 2.3)

In any case, I think that it is better to create another PR because it is different from the subject of PR (jruby support)

Reply:
I mean soft deprecation. Too many test cases waste travis's resources.